asmcli/asmcli.sh

```diff
@@ -51,7 +51,7 @@ MANAGED_WEBHOOKS=""
 EXPOSE_ISTIOD_SERVICE=""
 CANONICAL_CONTROLLER_MANIFEST=""
 
-SCRIPT_NAME="${0##*/}"
+SCRIPT_NAME=""
```

Comment:
I forgot if I asked this already or not, but can we just make this

```
SCRIPT_NAME="${0##*/}"; readonly SCRIPT_NAME
```

instead of setting it later?

Reply:
Done